source/crud/update.txt

```diff
@@ -147,42 +133,47 @@ the method selects and updates the first matched document. If no
 documents match the query filter, the update operation makes no
 changes. 
 
-See our :ref:`upsert guide <golang-upsert-guide>`
+See the :ref:`upsert guide <golang-upsert-guide>`
 to learn how to insert a new document if no documents match the query filter.
 
-Example
-```````
+UpdateOne() Example
```

Comment:
For technical reviewer: Can you please review this new UpdateOne() Ex? Thank you!

Reply:
👍 LGTM